shared/src/commonMain/kotlin/org/hyperskill/app/projects/domain/model/ProjectProgress.kt

```diff
@@ -23,7 +23,9 @@ data class ProjectProgress(
     @SerialName("feature_score")
     val featureScore: Float = 0f,
     @SerialName("base_score")
-    val baseScore: Float = 0f
+    val baseScore: Float = 0f,
+    @SerialName("completed_stages")
+    val completedStages: List<Long>?
```

Comment:
It's better to define an emptyList as a default value

Reply:
In this case we won't be able to differ situation when backend returned us null and empty list, is it ok?

Reply:
@XanderZhu yes, I agree. Will use these approach to fields I added/modified